components/SponsorsBar.tsx

```diff
@@ -9,6 +9,9 @@ export const SponsorsBar = () => (
       <Link href="https://lucavall.in" className="grayscale hover:grayscale-0">
         <Image src="/sponsors/lucavallin.png" alt="lucavallin" width={25} height={25} />
       </Link>
+      <Link href="https://chat.collectivai.com/" className="hover:scale-110">
```

Comment:
let's make sure all sponsor icons follow the same style:

- gray
- color &bigger when hover (but it should not move the page)